spec/sidekiq_unique_jobs/lock_config_spec.rb

```diff
@@ -88,9 +88,9 @@
     it { is_expected.to eq(:log) }
 
     context "when on_conflict is a hash" do
-      let(:on_conflict) { { client: :replace, server: :reschedule } }
+      let(:on_conflict) { { "client" => "replace", "server" => "reschedule" } }
 
-      it { is_expected.to eq(:reschedule) }
+      it { is_expected.to eq("reschedule") }
```

Comment:
It doesn't matter if this is a string or a symbol because it is always converted to a symbol later anyway : https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/lib/sidekiq_unique_jobs/on_conflict.rb#L34